R/plot_missing.r

```diff
@@ -26,11 +26,23 @@
 
 plot_missing <- function(data,
                          group = list("Good" = 0.05, "OK" = 0.4, "Bad" = 0.8, "Remove" = 1),
+                         fill_color = c("Good"   = "#1B9E77",
+                                        "OK"     = "#E6AB02",
+                                        "Bad"    = "#D95F02",
+                                        "Remove" = "#E41A1C"),
                          missing_only = FALSE,
                          geom_label_args = list(),
                          title = NULL,
                          ggtheme = theme_gray(),
                          theme_config = list("legend.position" = c("bottom"))) {
+  ## Verify group = fill_color
+  if (length(group) != length(fill_color)) {
```

Comment:
I have mixed feelings about these. This is what I think the update should include:

1. When using default, I am fine applying a hard-coded color template.
2. When not using default (not 4 groups), the color template should be automated instead of throwing an error.
3. End user should be able to define a customized color template.

I believe we have 1 and 3, but might need to tweak for 2.

Reply:
Done!